.stylelintrc.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "stylelint-config-standard"
-}
```

Comment:
No more stylelint?

Reply:
`prettier` [styles all the thingz](https://prettier.io/docs/en/index.html)

Reply:
Style lint is *also* not just formatting 😃 

It’s ok though

Reply:
If I have to deal with another permutation of tooling at this point I'm gonna start pulling the wings off of butterflies.  🤯

Reply:
Made a brief attempt at adding styleint back in (using `stylelint-prettier` and `stylelint-config-prettier`), but gave up when it started bitching about "When linting something other than CSS, you should install an appropriate syntax, e.g. "postcss-scss", and use the "customSyntax" option" after I'd told it to not lint the `.tsx` files.

I'm going to call this good for now.